doc_src/functional_hazard_analysis_longline/fha_longline.tex

```diff
@@ -355,34 +358,847 @@ \subsection{Phase 1: Approach and Positioning}
+	       	Furthermore, the downwash can cause the loops of the highline backup to swing around and get caught in various places on the \textcolor{purple}{patient} or \textcolor{teal}{rescuer}.
+	        These could have to be cut during the subsequent rescue process or could lead to unwanted tied-up situations.}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Test for upward-swinging behaviour already in phase 1 (approach)\\
+	     	\tabitem Sheathing of line and backup to prevent loops},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{orange}{medium}, as the formation of loops is prevented and a swinging up in phase 1 can be tested by slow approach}]
+
+%2.7
+\fhaTable[fhaSystemKeyValue=
+			{Highline},
+		 fhaFunctionKeyValue=
+		 	{load transfer},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{2.7},
+	     fhaCausesKeyValue=
+	     	{Rockfall or other objects blown down by downwash}, 
+	     fhaDescriptionKeyValue=
+	     	{A falling stone, branch, carabiner, etc. is enough to cut the line under tension.
+	      	As a result, the \textcolor{purple}{patient} suddenly falls into the sling and the line no longer supports his weight.
+	      	This can lead to a high steering load for the pilot.
+	      	The fall height is limited by the length of the connection between the rope and the patient belt}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Cleaning the stands of loose stones and objects that could be thrown up \\
+	     	\tabitem Prohibit actions near the highline that involve throwing/kicking off objects/etc. \\
+	     	\tabitem Minimize the possible fall height of the \textcolor{purple}{patient} in the event of a line tear},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{orange}{medium}. Very unlikely if the anchors are cleaned and personnel at the anchors are careful.
+			In addition, the fall height can be kept small and the pilot can be prepared}]
+
+%2.8
+\fhaTable[fhaSystemKeyValue=
+			{Leash},
+		 fhaFunctionKeyValue=
+		 	{load transfer},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{2.8},
+	     fhaCausesKeyValue=
+	     	{Fall of two people in leash when releasing the rope}, 
+	     fhaDescriptionKeyValue=
+	     	{If the rope has to be released during this phase, the \textcolor{teal}{rescuer} and \textcolor{purple}{patient} will fall into the patient's leash.}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Avoid sharp objects on harnesses/clothing of \textcolor{teal}{rescuer} and \textcolor{purple}{patient}\\
+	     	\tabitem Cutting tool for the leash cut remains secured in a holder\\
+	     	\tabitem No unprotected, sharp objects on \textcolor{purple}{patient} and \textcolor{teal}{rescuer}},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{red}{high}, but can easily be avoided by taking measures (use of suitable clothing and belts and appropriate caution).}]
+
+%2.9
+\fhaTable[fhaSystemKeyValue=
+			{belt},
```

Comment:
`Belt` is always wrong ... `harness` is meant ... should be safely doable with s'n'r

Reply:
done in https://github.com/JBludau/Highline-Rescue-Project/pull/10/commits/68759c11381d636bab6acabab3c5850506ac348f for the whole document